meeting-notes/2019/Q1/2019-01-02--gui-and-in-web-browsers-weekly.md

```diff
@@ -0,0 +1,115 @@
+    -   Added a warning about mixed-content when gateway URL is not a Secure Context - [ipfs-companion/pull/650](https://github.com/ipfs-shipyard/ipfs-companion/pull/650) 
+    -   Started discussion about the future of DNSLink cache - [ipfs-companion/pull/646](https://github.com/ipfs-shipyard/ipfs-companion/pull/646#issuecomment-450879014)  + [go-ipfs/issues/5884](https://github.com/ipfs/go-ipfs/issues/5884) 
+-   Next:
+    -   Finalize OKRs
+    -   tackle holiday backlog
+    -   release Companion
+
+  
+
+
+@hacdias
+
+-   Done:
+    -   Fixed issue where we were calling methods that only exist on macOS: <https://github.com/ipfs-shipyard/ipfs-desktop/pull/761>
+    -   Fixed the links not opening in external windows: <https://github.com/ipfs-shipyard/ipfs-desktop/pull/769>
+    -   Updated to Electron 4: <https://github.com/ipfs-shipyard/ipfs-desktop/pull/769>
+    -   Fixed the black screenshot issue: <https://github.com/ipfs-shipyard/ipfs-desktop/pull/769>
+    -   Improved the screenshots experience: there will be now a notification telling the user a screenshot was taken and if they click on it, Web UI will open on that screenshots. If the user has a multi-monitor setup, the screenshots will be in a directory.
+        -   <https://github.com/ipfs-shipyard/ipfs-desktop/pull/759>
+        -   <https://github.com/ipfs-shipyard/ipfs-desktop/pull/775>
+-   Blocked:
+    -   Waiting for macOS guys on this PR: <https://github.com/ipfs-shipyard/ipfs-desktop/pull/770>
+-   Next:
+    -   Go through Desktop’s issue list and see what can be closed and fixed.
+    -   Prioritize native OS integrations: <https://github.com/ipfs-shipyard/ipfs-desktop/issues/679>
+    -   Work on that ^^
+    -   🧙‍♂️
+
+@alanshaw
+-   Done:
+    -   JS IPFS 0.34 release dance 💃
+    -   Holidays
+-   Blocked:
+    -   JS IPFS 0.34 release blocked on stack overflow issues with latest libp2p
+-   Next:
+    -   Cool stuff coming to a JS IPFS near you:
+        -   Way smaller bundle is coming <https://github.com/ipfs/js-ipfs/pull/1795> minified+gzip >50% smaller!
+        -   DHT is coming! <https://github.com/ipfs/js-ipfs/pull/856> APPROVED - blocked on test failures
+        -   Fast `ipfs.add` is coming <https://github.com/ipfs/js-ipfs-unixfs-importer/pull/10> 
+            -  > In real-world use, this changed the time it takes to \`jsipfs add\` a 260MB file to a fresh repo from 13.7s to 1.95s. By comparison\`go-ipfs\` takes 1.58s to add the same file to a fresh repo.
+    -   Work on benchmarks for <https://benchmarks.ipfs.team>
+
+@terichadbourne
+
+-   Done:
+    -   \[Live] Restructured ProtoSchool website to include community info <https://github.com/ProtoSchool/protoschool.github.io/pull/103>
+    -   \[Merge pending review] Added a text-based tutorial on underlying concepts <https://github.com/ProtoSchool/protoschool.github.io/pull/59>
+-   Next:
+    -   Seek volunteers to lead chapters so there’s more content at launch
```

Comment:
@terichadbourne suggestion: What about pinging the folks that already organize IPFS Meetups all over the place?

Reply:
@daviddias That's in my plans to tackle after launch. For now I'm focusing on PL folks who can serve as guinea pigs for my chapter setup process before I reach out more broadly. :)

Reply:
The London crew has already stepped up. 🎉 A Porto chapter would be awesome if the folks there have the time and interest!

Reply:
hey hey, let's chat! How can we get the conversation going? Maybe schedule a call?